projects/plugins/jetpack/_inc/client/at-a-glance/security-bundle/index.jsx

```diff
@@ -0,0 +1,131 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getRedirectUrl } from '@automattic/jetpack-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import Button from 'components/button';
+import Card from 'components/card';
+import bundleImageUrl from './shield-with-check.svg';
+import cloudImageUrl from './cloud.svg';
+import shieldImageUrl from './shield.svg';
+import removeBugImageUrl from './remove-bug.svg';
+import { getProductDescriptionUrl } from 'product-descriptions/utils';
+import { isFetchingSitePurchases as getIsFetchingSitePurchases, hasActiveSecurityPurchase as getHasActiveSecurityPurchase } from 'state/site';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class DashSecurityBundle extends Component {
+	trackUpgradeClick = () => {
+		analytics.tracks.recordJetpackClick( {
+			page: 'aag',
+			target: 'upgrade-button',
+			feature: 'security',
+		} );
+	};
+
+	renderFeatures() {
+		const features = [
+			{
+				id: 'backup',
+				imageUrl: cloudImageUrl,
+				text: __( 'Backup', 'jetpack' ),
+				imgAlt: __( 'A cloud representing Jetpack Backup', 'jetpack' ),
+			},
+			{
+				id: 'scan',
+				imageUrl: shieldImageUrl,
+				text: __( 'Scan', 'jetpack' ),
+				imgAlt: __( 'A shield representing Jetpack Scan', 'jetpack' ),
+			},
+			{
+				id: 'antispam',
+				imageUrl: removeBugImageUrl,
+				text: __( 'Anti-spam', 'jetpack' ),
+				imgAlt: __( 'A crossed-out bug representing Jetpack Anti-spam', 'jetpack' ),
+			},
+		];
+
+		return (
+			<div className="dash-security-bundle--content-info-features">
+				{ features.map( ( { id, imageUrl, text, imgAlt } ) => (
+					<div id={ id } className="dash-security-bundle--content-info-features-single-feature">
+						<img src={ imageUrl } alt={ imgAlt } />
+						<p>{ text }</p>
+					</div>
+				) ) }
+			</div>
+		);
+	}
+
+	render() {
+		const { hasActiveSecurityPurchase, isFetchingSitePurchases } = this.props;
+
+		if ( hasActiveSecurityPurchase || isFetchingSitePurchases ) {
+			return null;
+		}
+
+		return (
+			<Card className="dash-security-bundle">
+				<div className="dash-security-bundle--icon">
+					<img src={ bundleImageUrl } alt={ __( 'A shield and check mark representing the Jetpack Security Bundle', 'jetpack' ) } />
+				</div>
+				<div className="dash-security-bundle--content">
+					<div className="dash-security-bundle--content-info">
+						<h2>{ __( 'Comprehensive Site Security', 'jetpack' ) }</h2>
```

Comment:
Should this be an h3 and stand out  a bit less, since that's part of the "Security" section that's labeled as an h2?

![image](https://user-images.githubusercontent.com/426388/146038720-538327dd-2c71-40da-9f10-1c4d6c6d7683.png)